### ISSUE TITLE
No profile in Tensorboard Profiler tab

### ISSUE THREAD

**Issue by dbl001** [tensorflow/profiler#346]
I am running tensorflow 2.5 from tensorflow-macos with tensorflow-metal on Big Sur 11.5.
The model trains and run, the GPU runs, Tensorboard is working properly with the exception of the profiler tab which is empty.

I did get this error message:
```
W0726 08:25:03.846074 123145487446016 application.py:556] path /data/index.js not found, sending 404
```
Here's my virtual environment;
```
% pip list
Package                    Version
-------------------------- -------------------
absl-py                    0.12.0
anyio                      3.2.1
appnope                    0.1.2
argon2-cffi                20.1.0
astunparse                 1.6.3
async-generator            1.10
attrs                      21.2.0
Babel                      2.9.1
backcall                   0.2.0
bleach                     3.3.1
cachetools        
I opened a Stack Overflow issue but haven't gotten any responses:

https://stackoverflow.com/questions/68532951/how-do-i-get-the-tensorflow-profiler-working-in-tensorflow-2-5-with-tensorflow

```
% pip list
Package                    Version
-------------------------- -------------------
absl-py                    0.12.0
anyio                      3.2.1
appnope                    0.1.2
argon2-cffi                20.1.0
astunparse                 1.6.3
async-generator            1.10
attrs                      21.2.0
Babel                      2.9.1
backcall                   0.2.0
bleach                     3.3.1
cachetools                 4.2.2
certifi                    2021.5.30
cffi                       1.14.6
charset-normalizer         2.0.1
cycler                     0.10.0
Cython                     0.29.24
debugpy                    1.3.0
decorator                  5.0.9
defusedxml                 0.7.1
dill                       0.3.4
dotmap                     1.3.23
entrypoints                0.3
flatbuffers                1.12
future                     0.18.2
gast                       0.4.0
gensim                     4.0.1
google-auth                1.32.1
google-auth-oauthlib       0.4.4
google-pasta               0.2.0
googleapis-common-protos   1.53.0
grpcio                     1.34.1
gviz-api                   1.9.0
h5py                       3.1.0
idna                       3.2
importlib-resources        5.2.0
ipykernel                  6.0.1
ipython                    7.25.0
ipython-genutils           0.2.0
ipywidgets                 7.6.3
jedi                       0.18.0
Jinja2                     3.0.1
json5                      0.9.6
jsonschema                 3.2.0
jupyter-client             6.1.12
jupyter-core               4.7.1
jupyter-server             1.9.0
jupyterlab                 3.0.16
jupyterlab-pygments        0.1.2
jupyterlab-server          2.6.1
jupyterlab-widgets         1.0.0
keras-nightly              2.5.0.dev2021032900
Keras-Preprocessing        1.1.2
kiwisolver                 1.3.1
Markdown                   3.3.4
MarkupSafe                 2.0.1
matplotlib                 3.4.2
matplotlib-inline          0.1.2
mistune                    0.8.4
nbclassic                  0.3.1
nbclient                   0.5.3
nbconvert                  6.1.0
nbformat                   5.1.3
nest-asyncio               1.5.1
notebook                   6.4.0
numpy                      1.19.5
oauthlib                   3.1.1
opt-einsum                 3.3.0
packaging                  21.0
pandas                     1.3.0
pandocfilters              1.4.3
parso                      0.8.2
pexpect                    4.8.0
pickleshare                0.7.5
Pillow                     8.3.1
pip                        21.1.3
prometheus-client          0.11.0
promise                    2.3
prompt-toolkit             3.0.19
protobuf                   3.17.3
ptyprocess                 0.7.0
pyasn1                     0.4.8
pyasn1-modules             0.2.8
pybind11                   2.6.2
pycparser                  2.20
Pygments                   2.9.0
pyparsing                  2.4.7
pyrsistent                 0.18.0
python-dateutil            2.8.2
pytz                       2021.1
pyzmq                      22.1.0
requests                   2.26.0
requests-oauthlib          1.3.0
requests-unixsocket        0.2.0
rsa                        4.7.2
scipy                      1.7.0
Send2Trash                 1.7.1
setuptools                 41.2.0
six                        1.15.0
smart-open                 5.1.0
sniffio                    1.2.0
tensorboard                2.5.0
tensorboard-data-server    0.6.1
tensorboard-plugin-profile 2.4.0
tensorboard-plugin-wit     1.8.0
tensorflow-datasets        4.3.0
tensorflow-estimator       2.5.0
tensorflow-hub             0.12.0
tensorflow-macos           2.5.0
tensorflow-metadata        1.1.0
tensorflow-metal           0.1.1
termcolor                  1.1.0
terminado                  0.10.1
testpath                   0.5.0
tornado                    6.1
tqdm                       4.61.2
traitlets                  5.0.5
typing-extensions          3.7.4.3
urllib3                    1.26.6
wcwidth                    0.2.5
webencodings               0.5.1
websocket-client           1.1.0
Werkzeug                   2.0.1
wheel                      0.36.2
widgetsnbextension         3.5.1
wrapt                      1.12.1
zipp                       3.5.0
<img width="2560" alt="Screen Shot 2021-07-26 at 8 41 17 AM" src="https://user-images.githubusercontent.com/3105499/127186791-6b44aa60-4d76-464d-a80d-a9b102a725b4.png">
<img width="2560" alt="Screen Shot 2021-07-26 at 8 40 59 AM" src="https://user-images.githubusercontent.com/3105499/127187069-4d4003eb-00c9-4f8a-a1a7-b8f2d77e1df6.png">
![Uploading Screen Shot 2021-07-26 at 8.40.53 AM.png…]()

```
